### PR TITLE
Making the emoji screen work with the key customisation, fixes #483

### LINKED ISSUE #483: emoji view changes keyboard height
**Thumb-Key Version**
2.3.0

**Describe the bug**
with the new key border setting, the emoji picker's built-in border makes the kebyboard taller than in any of the other layers. you can even see it on key border:1
this makes the keyboard jump up and down when going from and to emoji picker, which is annoying

**To Reproduce**
1. with key border enabled, go to emoji picker & see

### PR DESCRIPTION
**Pull request by WadeWT**
It looks a bit messy because I wrapped the emoji screen in a box, but I think it's all working nicely now.

### CODE REVIEW

File: app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardKey.kt [hunk 1]
```diff
@@ -78,7 +78,7 @@ fun KeyboardKey(
     capsLock: Boolean,
     legendSize: Int,
     keyPadding: Int,
-    keyBorderWidth: Int,
+    keyBorderWidth: Float,
```
Comment (dessalines) [inline app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardKey.kt:81]: Why does this need to change to a float, its an integer in the DB. You could just convert it to a float where necessary.

Comment (WadeWT) [reply app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardKey.kt:81]: Right you are, not quite sure what I was doing.

Comment (dessalines) [reply app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardKey.kt]: No probs!

**Review by dessalines — APPROVED** [review]
Tests great, thx!

**Pull request merged**